WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2851,16 +2854,18 @@ function(url) {
 /**
  * @private
  */
-ZmZimbraMail.standardHtmlLinkCallback =
+ZmZimbraMail.modernClientLinkCallback =
 function() {
 	var urlParams = {
 		path: appContextPath,
 		qsArgs: {
-			client: "standard"
+			client: "modern"
 		}
 	};
 	var url = AjxUtil.formatUrl(urlParams);
-	ZmZimbraMail.sendRedirect(url);	// will trigger onbeforeunload
+	if (url) {
```

Comment:
In Classic

- username dropdown contains "Modern Web App" option
- reroutes to Modern Web Client

Reply:
shardool has added a comment in the ticket 
https://jira.corp.synacor.com/browse/PREAPPS-4538?focusedCommentId=1330394&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1330394